apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -29,12 +32,22 @@ type Properties<T> =
 const Table = <T extends object>({
 	columns,
 	data,
+	isLoading,
 	...selectableProperties
 }: Properties<T>): JSX.Element => {
 	const { getRowId, onRowSelect, selectedRowIds } = selectableProperties as
 		| Record<keyof SelectableProperties<T>, undefined>
 		| SelectableProperties<T>;
 
+	const tbodyReference = useRef<HTMLTableSectionElement>(null);
+	const [tableBodyHeight, setTableBodyHeight] = useState<null | number>(null);
+
+	useEffect(() => {
+		if (!isLoading && tbodyReference.current) {
+			setTableBodyHeight(tbodyReference.current.clientHeight);
+		}
+	}, [isLoading]);
```

Comment:
This logic looks too complicated for its gain. Instead, let's just set min-height for table-data instead of max to make it fixed. I think even if we have fewer rows (for example, we are on the last page) we still want the table to have the same height so that pagination doesn't jump

Reply:
But I can't set a fixed height, then the table will just jump. Because if you select 5 elements in the pagination, it will be one height, if 10 is another. Because of this I dynamically get the height of the table so that the loader has the same height.
I still don't understand how the min-height will make the table not jump. What exactly should be the min-height?

Reply:
I mean I see two possible ways of implementation:

1. Set a fixed min-size for the table during loading, for example, min-height 300px. And this width doesn't depend on the table page size.
2. Set a fixed min-height for the table overall instead of max-height. This means if we don't have enough rows on a page we will display empty space, but the height will remain the same. 

take a look at this example, on the last table page
https://mui.com/base-ui/react-table-pagination/#customized-look-and-feel